packages/amesos2/src/Amesos2_Superludist_def.hpp

```diff
@@ -1007,6 +1007,10 @@ namespace Amesos2 {
       // reinex GIDs
       superlu_rowmap_ = this->matrixA_->getRowMap(); // use original map to redistribute vectors in solve
       Teuchos::RCP<const MatrixAdapter<Matrix> > contig_mat = this->matrixA_->reindex(superlu_contig_rowmap_, superlu_contig_colmap_, current_phase);
+
+      superlu_contig_rowmap_ = contig_mat->getRowMap();
+      superlu_contig_colmap_ = contig_mat->getColMap();
+
```

Comment:
@maxfirmbach. Were these needed? I thought these get initialized inside `reindex`?

Reply:
Ah, thank you for the fix, @maxfirmbach!! I think this is better. Also, it looks like I did not test it well with `epetra`. So, thank you for these testers, too!!